searchlib/src/tests/features/prod_features.cpp

```diff
@@ -1563,6 +1565,26 @@ Test::testMatchCount()
     }
 }
 
+void
+Test::testUnique()
+{
+    {
+        UniqueBlueprint bp;
+        EXPECT_TRUE(assertCreateInstance(bp, "unique"));
+        FtFeatureTest ft(_factory, "");
+        StringList params, in, out;
+        FT_SETUP_OK(bp, ft.getIndexEnv(), params, in, out.add("out"));
+        FT_DUMP_EMPTY(_factory, "unique");
+
+        EXPECT_TRUE(assertMatches(0, "x", "a", "matches(foo)"));
```

Comment:
Should this line be here?

Reply:
Thanks, leftover from copy-paste-mutate.